src/views/DataExploration/components/UploadIds/ParticipantUploadIds.tsx

```diff
@@ -22,8 +22,8 @@ const ParticipantUploadIds = ({ queryBuilderId }: OwnProps) => (
   <EntityUploadIds
     entityId="participant"
     entityIdTrans="Participant"
-    entityIdentifiers="Participant ID"
-    placeHolder="e.g. pt-005X8BR9"
+    entityIdentifiers="Participant ID, Participant External ID"
```

Comment:
Dans le ticket c'est écrit `Search by Entity ID, External Entity ID` donc ça devrait être
```suggestion
    entityIdentifiers="Participant ID, External Participant ID"
```

Reply:
Done!